package.json

```diff
@@ -40,6 +40,7 @@
     "bootstrap": "^3.4.1",
     "bootstrap-colorpicker": "^2.5.3",
     "buffer": "~5.4.0",
+    "canvas": "^2.6.1",
```

Comment:
Why was this added?

Reply:
If I remeber correctly It was added to fix canvas resize but later I fixed it without it so we can remove it now